generation/scrapePgnData.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { updatePgnData, scrapePgnPage, pgnRootPage } from "./scrapePgnData"
+
+describe('PGN Data Scraper', () => {
+
+    it('should return data', async () => {
+        const pgns = await scrapePgnPage(pgnRootPage)
```

Comment:
@oliverrahner I'd prefer to avoid any HTTP requests in unit tests. It decreases stability of CI/CD pipelines. Maybe it makes sense to use a mock here?